File: 01-input-output/06-average-three-numbers/solution.c
#include <stdio.h>


float get_number(const char *label);


int main(void){
    const char *first = "first";
    const char *second = "second";
    const char *third = "third";

    float first_number = get_number(first);
    float second_number = get_number(second);
    float third_number = get_number(third);

    float sum = first_number + second_number + third_number;

    float average = sum/3;

    printf("Average of %f, %f, %f is %f\n", first_number, second_number, third_number, average);
    return 0;
}


float get_number(const char *label){
    printf("Enter %s number : ", label);
    float number;
    scanf("%f", &number);
    return number;
}

File: 01-input-output/06-average-three-numbers/solution.js
import readline from "readline/promises";
import {stdin as input, stdout as output} from 'process';

const rl = readline.createInterface({
    input, output
});


let firstNumber = (await rl.question("Enter first number : ")).trim();
let secondNumber = (await rl.question("Enter second number : ")).trim();
let thirdNumber = (await rl.question("Enter third number : ")).trim();

firstNumber = parseFloat(firstNumber);
secondNumber = parseFloat(secondNumber);
thirdNumber = parseFloat(thirdNumber);

let average = (firstNumber + secondNumber + thirdNumber) / 3

console.log(`Average of ${firstNumber}, ${secondNumber}, ${thirdNumber} is ${average}`)
rl.close()